src/main.py

```diff
@@ -17,7 +14,7 @@ def main():
     preset_name = sys.argv[1]
     nodes, curr_edges, routes, vehicles = [], [], [], []
     
-    manager = load_preset(preset_name, nodes, curr_edges, routes, vehicles)
+    manager, nodes, curr_edges, routes, vehicles = load_preset(preset_name, nodes, curr_edges, routes, vehicles)
```

Comment:
so the new format is:
manager, nodes, curr_edges, routes, vehicles = load_preset(preset_name)

instead of passing empty lists and populating them, load_preset would return lists, and

manager, nodes, curr_edges, routes, vehicles will be defined from the return of load_preset